prow/scripts/e2e-test/reconcile-kyma.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
```

Comment:
Do we need this script in test-infra repo as well? We have this script in reconciler repo, which we are executing in the e2e test.

Reply:
no, removed